.github/workflows/ci-model-regression.yml

```diff
@@ -604,6 +622,10 @@ jobs:
           DATASET_REPOSITORY_BRANCH: ${{ matrix.dataset_branch }}
           TYPE: ${{ matrix.type }}
           DATASET_COMMIT: ${{ steps.set-dataset-commit.outputs.dataset_commit }}
+          BRANCH: ${{ steps.extract_branch.outputs.branch }}
+          DD_ENV: regression-tests # ${{ matrix.test }}
```

Comment:
How come matrix.test is commented?

Reply:
It looks like it's copy & pastes from the other workflow where `${{ matrix.text }}` makes sense. Here a constant name is ok.

I'd change the name to `rasa-regression-tests`.